package.json

```diff
@@ -105,9 +106,11 @@
     ]
   },
   "devDependencies": {
+    "@types/node": "^6.0.36",
```

Comment:
Note that because we're not using a real strong compiler, typings are kind of ignored. I want to fix that at some point soon.